cloudinit/config/cc_ubuntu_advantage.py

```diff
@@ -208,20 +213,36 @@ def configure_ua(token=None, enable=None, config=None):
             error=str(e)
         )
         util.logexc(LOG, msg)
-        raise RuntimeError(msg) from e
+
+        if not re.search("This machine is already attached to", str(e)):
+            raise RuntimeError(msg) from e
+
     enable_errors = []
+    enable_warnings = []
```

Comment:
Thinking more about this, wouldn't be better to just log this with DEBUG level? We are handling this case and there is nothing to justify it as WARNING, just to improve debug-ability.

See the main message, containing a diff proposal.